### ISSUE TITLE
Suggestion: Another way to avoid debugging in a single process

### ISSUE THREAD

**Issue by mauler2025** [PacktPublishing/Elevating-Game-Experiences-with-Unreal-Engine-5-Second-Edition#1]
Chapter 16:

>The reason why the `RepNotify` message is showing in the Server window instead of the Client window is that, when playing in the editor, both windows share the same process, so printing text on the screen won’t be accurate. To get the correct behavior, you’ll need to run the packaged version of the game.

My solution instead of packaging the game during development iterations, we just need to  uncheck `Run Under One Process` from `Advanced Settings... | Multiplayer Options`.

**Comment by xlar8or** [comment]
Yes that is correct, that text was used straight from the unreal engine 4
version of the book, which didn't have that option at the time.
